gratipay/models/package/__init__.py

```diff
@@ -40,3 +51,35 @@ def from_names(cls, package_manager, name):
         """
         return cls.db.one("SELECT packages.*::packages FROM packages "
                           "WHERE package_manager=%s and name=%s", (package_manager, name))
+
+
+    @property
+    def team(self):
+        """A computed attribute, the :py:class:`~gratipay.models.team.Team`
+        linked to this package if there is one, otherwise ``None``. Makes a
+        database call.
+        """
+        return self._load_team(self.db)
+
+
+    def ensure_team(self, cursor, owner):
+        """Given a db cursor and :py:class:`Participant`, insert into ``teams`` if need be.
+        """
+        team = self._load_team(cursor)
```

Comment:
For comparison, here is the output from a simpler thread target clearly showing parallelism:

```
45091921924513398784

4509192192
4513398784
45133987844509192192

4509192192
4513398784
4509192192
4513398784
45091921924513398784

45091921924513398784

4509192192
4513398784
4513398784
4509192192
4509192192
4513398784
F

================================================== FAILURES ==================================================
______________________ PackageLinking.test_finishing_email_verification_is_thread_safe _______________________
Traceback (most recent call last):
  File "/Users/whit537/personal/gratipay/gratipay.com/tests/py/test_email.py", line 811, in test_finishing_email_verification_is_thread_safe
    assert results[a.ident] == _email.VERIFICATION_SUCCEEDED
KeyError: 4509192192
```

```diff
diff --git a/tests/py/test_email.py b/tests/py/test_email.py
index fd5280f..d2928ec 100644
--- a/tests/py/test_email.py
+++ b/tests/py/test_email.py
@@ -778,9 +778,12 @@ class PackageLinking(VerificationBase):
         nonce = self.alice.get_email(self.address).nonce
 
         results = {}
+        import time
         def finish():
             key = threading.current_thread().ident
-            results[key] = self.alice.finish_email_verification(self.address, nonce)
+            for i in range(10):
+                print(key)
+                time.sleep(0.1)
 
         def t():
             t = threading.Thread(target=finish)
```